VAMobile/src/components/FormWrapper/FormFields/Picker/VAModalPicker.test.tsx

```diff
@@ -107,33 +103,31 @@ context('VAModalPicker', () => {
 
   describe('when labelKey exists', () => {
     it('should render a textview for the label', async () => {
-      const textViews = testInstance.findAllByType(TextView)
-      expect(textViews[7].props.children).toEqual(['Number', ' ', ''])
-      expect(textViews.length).toEqual(9)
+      expect(component.getByTestId('Number picker')).toBeTruthy()
```

Comment:
update to screen.getByTestId

Reply:
The testId was changed, so now I getByRole('spinbutton', { name: 'Back picker Navigates to the previous page' }) and that works to get the correct element.